mysql-test/t/percona_bug_ps3919-master.opt

```diff
@@ -0,0 +1 @@
+--log_bin=binlog --gtid_mode=ON --enforce_gtid_consistency=ON --sql_mode=ONLY_FULL_GROUP_BY
```

Comment:
Why does it need sql_mode?

Reply:
I'm not sure - but without that parameter, the crash didn't happen. The original bug report mentioned several other parameters, and I was able to remove most of them, but not this one.